ElectronClient/app/gui/NoteText.jsx

```diff
@@ -1428,7 +1439,13 @@ class NoteTextComponent extends React.Component {
 			const s1 = this.state.note.body.substr(0, selection.start);
 			const s2 = replacementText ? replacementText : this.state.note.body.substr(selection.start, selection.end - selection.start);
 			const s3 = this.state.note.body.substr(selection.end);
-			newBody = s1 + string1 + s2 + string2 + s3;
+
+			newBody = s1;
```

Comment:
s1, s2 and s3 are used only in this small context. Please consider a renaming of the variables.
s1 is the preceding line
s2 is the body
s3 is the next line

Reply:
Variables s1 and s3 are redundant and so eliminated. s2 renamed.